test/disabled.html

```diff
@@ -0,0 +1,85 @@
+<!doctype html>
+
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>iron-list test</title>
```

Comment:
nit: the title is wrong.

Reply:
Done. Also opened a chore PR fixing the title of other test pages: https://github.com/vaadin/vaadin-grid/pull/2212